PhysicsTools/HeppyCore/python/framework/eventsfwlite.py

```diff
@@ -15,16 +18,19 @@
 
 class Events(object):
     def __init__(self, files, tree_name,  options=None):
-	if options is not None :
-		if not hasattr(options,"inputFiles"):
-		 	options.inputFiles=files
-		if not hasattr(options,"maxEvents"):
-			options.maxEvents = 0	
-		if not hasattr(options,"secondaryInputFiles"):
-			options.secondaryInputFiles = []
-	        self.events = FWLiteEvents(options=options)
+        logging.info('opening input files:')
+        logging.info(pprint.pformat(files))
```

Comment:
shouldn't they be sent in just one message, e.g. `'opening input files: %s' % pprint.pformat(files)` ?

Reply:
done